packages/contracts/test/registry/registry.ts

```diff
@@ -1,49 +0,0 @@
-import {expect} from 'chai';
-import {ethers} from 'hardhat';
-
-import {customError} from '../test-utils/custom-error-helper';
-
-const EVENTS = {
-  NewDAORegistered: 'NewDAORegistered',
-};
-
-describe('Registry', function () {
-  let registry: any;
-  let ownerAddress: string;
```

Comment:
We are removing this test file, but are we explicitly testing the base class with the new DAO registry and Plugin registry test counterparts?

Reply:
we are not removing here, it is renamed to `dao-registry.ts`